scripts/superdesk/editor2/editor.js

```diff
@@ -566,11 +566,23 @@ function SdTextEditorBlockEmbedController($timeout, editor, renditions, config)
             vm.model.caption = caption;
             // update the caption in the view
             vm.caption = caption;
+
             // on change callback
             $timeout(function() {
                 vm.onBlockChange();
             });
         },
+        handlePasta: function(e) {
```

Comment:
Mmmm finally, handling food with Superdesk :) I prefer meat, but pasta also works for me :)